app/templates/app/search.html

```diff
@@ -3,14 +3,82 @@
 
 {% block content %}
 
-<div class="body">
-    <div class="section mt-3 mb-3">
-        <div class="card content-card">
-            <div class="card-body">
-                <h5 class="card-title">Search a term</h5>
-                <p class="card-text text-center">
-                    Functionality of this page coming soon
-                </p>
+    <div class="body" xmlns="http://www.w3.org/1999/html">
```

Comment:
It looks as if there is some unnecessary changes here. If I'm looking correct, the lines from the body div to the if statement didn't need to move, and I guess the xmlns isn't needed. Please say if I'm missing something.